project/templates/configs/springboard.ini

```diff
@@ -8,7 +8,9 @@ pyramid.debug_routematch = false
 
 pyramid.default_locale_name = {{default_language}}
 
-unicore.content_repo_urls = {{ucd_host}}/repos/{{repo_name}}.json
+unicore.content_repo_urls =
```

Comment:
At the moment springboard and springboard-iogt don't differentiate. Can we add it in another PR? I'll have to update springboard first.